Tests/Unit/DependencyInjection/HttpClientExtensionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Http Client bundle.
+ *
+ * (c) David Buchmann <mail@davidbu.ch>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Http\ClientBundle\Tests\Unit\DependencyInjection;
+
+use FOS\HttpCacheBundle\DependencyInjection\FOSHttpCacheExtension;
```

Comment:
Not that I am complaining, but why is FOS HttpCacheBundle used here? Testing purpose or just copy&paste?